src/components/FeatureViews/Accounts/TabComponents/Schedule.js

```diff
@@ -1,5 +1,7 @@
 import { connect } from 'react-redux';
 import React, { Component } from 'react';
```

Comment:
If this is rewritten as a function component, the `{Component}` import from React can be removed.